build/build-sdk-images/node/gen.sh

```diff
@@ -24,15 +24,18 @@ sdk=/go/src/agones.dev/agones/proto/sdk
 googleapis=/go/src/agones.dev/agones/proto/googleapis
 gatewaygrpc=/go/src/agones.dev/agones/proto/grpc-gateway
 
-mkdir -p ./sdks/nodejs/lib/alpha
-
 cd /go/src/agones.dev/agones
 
+mkdir -p ./sdks/nodejs/lib/alpha
+mkdir -p ./sdks/nodejs/lib/beta
+
 grpc_tools_node_protoc -I ${googleapis} -I ${gatewaygrpc} -I ${sdk} --js_out=import_style=commonjs,binary:./sdks/nodejs/lib google/api/annotations.proto google/api/client.proto google/api/field_behavior.proto google/api/http.proto google/api/launch_stage.proto google/api/resource.proto protoc-gen-openapiv2/options/annotations.proto protoc-gen-openapiv2/options/openapiv2.proto
 grpc_tools_node_protoc -I ${googleapis} -I ${gatewaygrpc} -I ${sdk}/alpha --js_out=import_style=commonjs,binary:./sdks/nodejs/lib/alpha google/api/annotations.proto google/api/client.proto google/api/field_behavior.proto google/api/http.proto google/api/launch_stage.proto google/api/resource.proto protoc-gen-openapiv2/options/annotations.proto protoc-gen-openapiv2/options/openapiv2.proto
```

Comment:
As `client.proto` is not used `launch_stage.proto` is not needed
`field_behavior.proto` and `google/api/resource.proto` are also not needed

```suggestion
grpc_tools_node_protoc -I ${googleapis} -I ${gatewaygrpc} -I ${sdk}/alpha --js_out=import_style=commonjs,binary:./sdks/nodejs/lib/alpha google/api/annotations.proto google/api/http.proto protoc-gen-openapiv2/options/annotations.proto protoc-gen-openapiv2/options/openapiv2.proto
```

Reply:
Thanks for having a look ! I'm gonna apply your changes and will regenerate it, i'll do that asap